experimental/add2app/SimpleIOSExample/Podfile

```diff
@@ -0,0 +1,26 @@
+# Uncomment the next line to define a global platform for your project
+# platform :ios, '9.0'
+
+# Added these two lines for Flutter:
+flutter_application_path = '../example_module'
+system("pushd #{flutter_application_path} && flutter create . && popd")
```

Comment:
This line is surprising. What's the intent here?

Reply:
We have a thread in the Add-to-app chat about this.  The podhelper.rb script is in the .gitignore, so someone who just cloned this repo needs to create it or they can't `pod install`.  I don't really want it to be checked in, but maybe there's a better solution?

Reply:
Ah ok. Aren't these made by running flutter packages get instead of flutter create?

Reply:
Both work, apparently. Does `flutter create` also get packages?

Reply:
Strike that, actually. When I try with `flutter packages get`, the script is created, but there's some other part of the module missing. I end up with an error in Xcode that states it can't find `lib/main.dart`.

Reply:
What are the error outputs? Might be a valid bug.

Reply:
Weird. I can no longer replicate the failure.